src/main/java/org/opentest4j/MultipleFailuresError.java

```diff
@@ -90,11 +92,13 @@ private static String pluralize(int count, String singular, String plural) {
 		return count == 1 ? singular : plural;
 	}
 
-	private static String nullSafeMessage(AssertionError failure) {
+	private static String nullSafeMessage(Throwable failure) {
+		if (failure == null) {
```

Comment:
I work on JUnit 4 but not JUnit5, so take this with a grain of salt.

Taking a look at `TestExecutionResult`  it looks like the `Throwable` is always present for non-successful results. So it should be an error to pass in a `null` into this class.